internal/impl/config.go

```diff
@@ -157,3 +160,18 @@ type volumeSpecs struct {
 	Volumes      []corev1.Volume
 	VolumeMounts []corev1.VolumeMount
 }
+
+// telemetry contains options to control how the telemetry is being manipulated.
+type telemetry struct {
+	Metrics metricOpts
+}
+
+// metricOpts contains options to configure how the metrics are handled by the deployer.
+type metricOpts struct {
+	// If true, the deployer also exports metrics created by the framework; e.g.,
+	// autogenerated metrics, http metrics.
+	Generated bool
+
+	// How often to export the metrics.
```

Comment:
Do you want to mention that this defaults to thirty seconds?

Reply:
Good point.